builder/llpcBuilderRecorder.cpp

```diff
@@ -392,6 +422,191 @@ Instruction* BuilderRecorder::CreateReadClock(
     return Record(Opcode::ReadClock, getInt64Ty(), getInt1(realtime), instName);
 }
 
+
+// =====================================================================================================================
+// Create tan operation
+Value* BuilderRecorder::CreateTan(
+        Value*        pInValue,             // [in] Input value
+        const Twine&  instName)
+{
+    SmallVector<Value*, 2> args;
+    LLPC_ASSERT(pInValue != nullptr);
+    args.push_back(pInValue);
+    return Record(Tan, ( pInValue->getType()), args, instName, {});
```

Comment:
( pInValue->getType()) -> pInValue->getType()
Apply this to the following functions.

Reply:
We should be autogenerating all of this.